### PR TITLE
Fix eval, add tests for hocr-eval and hocr-eval-geom

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: hocr-eval [hunk 1]
```diff
@@ -44,7 +44,8 @@ def get_bbox(node):
     return tuple([int(x) for x in bbox.split()])
 
 def get_text(node):
-    s = node.text
+    textnodes = node.xpath(".//text()")
+    s = "".join([text for text in textnodes])
```
Comment (zuphilip) [inline hocr-eval]: This failed in Python 3. Therefore, I substitute this code with another occurrence of this function.

**Comment by zuphilip** [comment]
ping @stweil @kba

**Comment by kba** [comment]
Looks fine for me, the same `get_text` as in the other tools.

**Comment by stweil** [comment]
Thanks, @zuphilip.

**Pull request merged**